src/resolvers/Home.ts

```diff
@@ -1,4 +1,5 @@
 import { Context } from '../utils'
+const gql = s => s
```

Comment:
can you add a comment as to why this is needed, linking to an issue with more information?

Reply:
My bad. Replaced it with `import gql from graphql-tag`